### PR TITLE
[TRANSLATION] Update and fix Bulgarian (bg-BG) translation

### PR DESCRIPTION
**Pull request by delphipro**
## Purpose

_Do a quick recap of your work here._

JIRA issue: [CORE-XXXX](https://jira.reactos.org/browse/CORE-XXXX)

## Proposed changes

_Describe what you propose to change/add/fix with this pull request._

- 
- 

## TODO

_Use a TODO when your pull request is Work in Progress._

- [ ] 
- [ ]

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,10 +6,12 @@ modules/livecd_extras
...
 modules/[Aa][Hh][Kk]_[Tt]ests
+.vs
 .cache
 .cproject
 .DS_Store
 .project
 .settings
 .vscode
 sdk/tools/winesync/winesync.cfg
+out
```
Comment (binarymaster) [inline .gitignore:17]: Please revert unrelated changes here.

File: dll/cpl/access/lang/en-US.rc [hunk 2]
```diff
@@ -44,20 +44,20 @@ END
...
-    GROUPBOX "Cursor options", -1, 6, 79, 234, 115
+    GROUPBOX "Опции за курсора", -1, 6, 79, 234, 115
     LTEXT "Use the track bars to modify the blinking speed of the cursor and its width.",
           -1, 12, 88, 222, 20
-    CTEXT "Blinking s&peed:", -1, 12, 115, 222, 11
-    LTEXT "None", -1, 18, 130, 36, 11
-    LTEXT "Fast", -1, 192, 130, 36, 11
+    CTEXT "Скорост на премигване:", -1, 12, 115, 222, 11
+    LTEXT "Няма", -1, 18, 130, 36, 11
+    LTEXT "Бърза", -1, 192, 130, 36, 11
```
Comment (binarymaster) [inline dll/cpl/access/lang/en-US.rc:60]: This is en-US resource file, not bg-BG.

File: dll/cpl/sysdm/lang/bg-BG.rc [hunk 3]
```diff
@@ -348,12 +348,12 @@ BEGIN
     IDS_USERPROFILE_LOCAL "Local"
     IDS_USERPROFILE_ROAMING "Roaming"
     IDS_USERPROFILE_MANDATORY "Mandatory"
-    IDS_USERPROFILE_CONFIRM_DELETE "Are you sure you want to delete %s's profile?"
+    IDS_USERPROFILE_CONFIRM_DELETE "Сигурен ли сте че искате да изтриете този профил: %s's?"
```
Comment (binarymaster) [inline dll/cpl/sysdm/lang/bg-BG.rc:351]: The `'s` part at the end should probably be deleted.

File: dll/win32/browseui/lang/bg-BG.rc [hunk 4]
```diff
@@ -309,37 +309,37 @@ END
...
+    IDS_TOOLBAR_ERR_TEXT "Не може да се създаде лента с инструменти."
 END
 
 STRINGTABLE
 BEGIN
-    IDS_SEARCH_RESULTS "Search Results"
-    IDS_SEARCHINVALID "'%1' is not a valid folder."
-    IDS_SEARCH_FOLDER "Searching %s"
-    IDS_SEARCH_FILES_FOUND "%u files found"
+    IDS_SEARCH_RESULTS "Търсене. . ."
```
Comment (binarymaster) [inline dll/win32/browseui/lang/bg-BG.rc:331]: This looks incomplete.

File: media/inf/shortcuts.inf [hunk 5]
```diff
@@ -1,4 +1,4 @@
-﻿[Version]
+[Version]
```
Comment (binarymaster) [inline media/inf/shortcuts.inf:1]: Some unrelated change here, you probably inserted UTF-8 BOM.

**Review by binarymaster — DISMISSED** [review]

**Comment by binarymaster** [comment]
@delphipro the email you've used with commits in this PR is not associated with your GitHub account. Please fix that in the [email settings](https://github.com/settings/emails), or use different email for your commits.

File: .gitignore (hunk 1 above)
Comment (delphipro) [reply .gitignore]: Is reverted.

File: dll/cpl/access/lang/en-US.rc (hunk 2 above)
Comment (delphipro) [reply dll/cpl/access/lang/en-US.rc]: The file is reverted

File: dll/cpl/sysdm/lang/bg-BG.rc (hunk 3 above)
Comment (delphipro) [reply dll/cpl/sysdm/lang/bg-BG.rc]: 's removed

File: dll/win32/browseui/lang/bg-BG.rc (hunk 4 above)
Comment (delphipro) [reply dll/win32/browseui/lang/bg-BG.rc]: Fixed

File: media/inf/shortcuts.inf (hunk 5 above)
Comment (delphipro) [reply media/inf/shortcuts.inf]: Reverted

File: .gitignore (hunk 1 above)
Comment (delphipro) [reply .gitignore]: Why don't you add the .vs folder when working with Visual Studio?

**Review by binarymaster — COMMENTED** [review]
Ok my review has been addressed, however PR needs a reviewer, @HBelusca maybe? Also the point in this comment https://github.com/reactos/reactos/pull/5581#issuecomment-1683850756 is still standing.

File: .gitignore (hunk 1 above)
Comment (binarymaster) [reply .gitignore]: I asked to revert it because it's unrelated to the translation. But feel free to propose these changes in a separate PR.

**Comment by delphipro** [comment]
> Ok my review has been addressed, however PR needs a reviewer, @HBelusca maybe? Also the point in this comment [#5581 (comment)](https://github.com/reactos/reactos/pull/5581#issuecomment-1683850756) is still standing.

done

File: dll/win32/syssetup/lang/bg-BG.rc [hunk 6]
```diff
@@ -204,7 +204,7 @@ BEGIN
      *   samsrv.dll:IDS_USER_ADMINISTRATOR_NAME synchronized.
      *   Also check the IDD_COMPUTERPAGE dialog.
      */
-    IDS_ADMINISTRATOR_NAME "Administrator"
+    IDS_ADMINISTRATOR_NAME "Администратор"
```
Comment (HBelusca) [inline dll/win32/syssetup/lang/bg-BG.rc:207]:
```suggestion
    IDS_ADMINISTRATOR_NAME "Administrator" // FIXME: Should be "Администратор", but bg-BG translation missing in samsrv.dll
```

**Review by HBelusca — APPROVED** [review]
